outlook/calendar/pkg/printers/events.go

```diff
@@ -12,6 +12,28 @@ import (
 	"github.com/microsoftgraph/msgraph-sdk-go/models"
 )
 
+func EventToString(ctx context.Context, client *msgraphsdkgo.GraphServiceClient, calendar graph.CalendarInfo, event models.Eventable) string {
+	var calendarName string
+	if calendar.Owner == graph.OwnerTypeUser {
+		calendarName = util.Deref(calendar.Calendar.GetName())
+	} else {
+		groupName, err := graph.GetGroupNameFromID(ctx, client, calendar.ID)
+		if err != nil {
+			calendarName = calendar.ID
+		} else {
+			calendarName = groupName
+		}
+	}
+
+	var sb strings.Builder
+	sb.WriteString("Subject: " + util.Deref(event.GetSubject()) + "\n")
+	sb.WriteString("  ID: " + util.Deref(event.GetId()) + "\n")
+	sb.WriteString("  Start: " + util.Deref(event.GetStart().GetDateTime()) + "\n")
+	sb.WriteString("  End: " + util.Deref(event.GetEnd().GetDateTime()) + "\n")
+	sb.WriteString("  In calendar: " + calendarName + " (ID " + calendar.ID + ")\n")
```

Comment:
nit: I know it saves tokens to print it this way, but at some point I think minified JSON streams with `Share Context` containing a JSON schema of the output (along with descriptions) would be both more maintainable and give the LLM a better idea of the output, especially when we're dealing with structured data.

Reply:
I think I'll also do this in a follow-up PR. This is how we've been giving the data to the LLM and it has worked well enough for now. I want to get this in so that Darren can try it out and tell me what needs to change.